src/karapace/logging.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+
+from karapace.config import Config
+
+import logging
+import sys
+
+
+def configure_logging(*, config: Config) -> None:
```

Comment:
Even here, we can already use the DI, but let's move a bit and we build on this.